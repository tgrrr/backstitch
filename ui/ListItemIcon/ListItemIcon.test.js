import React from 'react'
import renderer from 'react-test-renderer'
import { ListItemIcon } from '.'

it('ListItemIcon: default', () => {
    const component = renderer.create(
        <ListItemIcon>
            <>
                ListItemIcon
            </>
        </ListItemIcon>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
