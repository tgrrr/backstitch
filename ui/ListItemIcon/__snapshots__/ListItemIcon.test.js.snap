// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItemIcon: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Listitemicon "
    data-testid="Listitemicon"
  >
    <div
      className="MuiListItemIcon-root"
    >
      ListItemIcon
    </div>
  </div>
</div>
`;
